boale/serverlessTestApp
# Task 2.2 Automated deployment

What was done:
Bucket created and distribution placed https://angular-shopping-project.s3.amazonaws.com/index.html
Permissions restricted to cloudfront only. https://d17979vaqi6lh.cloudfront.net
Configured automated client deploy, cloudformation setup, cloudformation invalidation using serverless

FE github: https://github.com/VladOstk/angular_aws_node_shop_pet

Note: in order to verify serverless solution:
1. checkout FE repo 
2. build via npm run build command
3. place dist assets inside build folder of serverless solution
4. configure yml provider.profile, custom.client.bucketname params to yours
5. deploy cloudformation stack via npm run cloudfront:deploy
6. deploy client assets via  npm run client:deploy

To verify cloudfront invalidation
1. Place updated files in serverless/build folder
2. run npm run client:deploy command
3. run npm run cloudfront:invalidateCache command
